### PR TITLE
Attach new csv to collective report

### PR DESCRIPTION
**Pull request by znarf**
Let's try to ship it today and gather some feedback.

### CODE REVIEW

File: cron/monthly/collective-report.js [hunk 1]
```diff
@@ -181,6 +185,20 @@ const processCollective = collective => {
         return collective;
       });
     })
+    .then(async collective => {
+      const transactionsCsvV2 = await getCollectiveTransactionsCsv(collective, { startDate, endDate });
+      if (transactionsCsvV2) {
+        const csvFilenameV2 = `${collective.slug}-${moment(d).format(dateFormat)}-transactions-v2.csv`;
+        options.attachments = [
```
Comment (kewitz) [inline cron/monthly/collective-report.js:192]: Actually, if we're including this a second report, we'll need to do:
```suggestion
        options.attachments.push(
```

**Review by kewitz — APPROVED** [review]
LGTM, we'll only need to make sure we're attaching both CSVs for now.

File: cron/monthly/collective-report.js (hunk 1 above)
Comment (znarf) [reply cron/monthly/collective-report.js]: Thank you for that. When testing locally, we can't see the attachments in MailDev.

Comment (kewitz) [reply cron/monthly/collective-report.js]: You can change the view to the Attachments tab:
Screenshot from 2021-08-31 16-03-39

**Pull request merged**